app/src/components/common/header/index.tsx

```diff
@@ -72,6 +77,11 @@ export const Header: React.FC = props => {
         <MobileMenuStyled toggleMenu={toggleMenu} isMenuOpen={isMenuOpen} />
         <ConnectedWeb3>
           <NetworkStyled />
+          <LogoutStyled
+            callback={() => {
+              setModalState(false)
```

Comment:
Is this necessary?

Reply:
Yes, because the state for the modal is true, and if you logout of the dapp then the modal is show really quickly in the screen . Is not so nice that you try to logout and the dapp display immediately the modal to login, any thoughts realted to this behaviour?

Reply:
That didn't happen to me, and I don't see why it would. But it's OK, I don't mind having this.